examples/multigpu_advtrain/target_attack.py

```diff
@@ -0,0 +1,257 @@
+import numpy as np
+import tensorflow as tf
+
+from cleverhans.model import Model, CallableModelWrapper
+from cleverhans.attacks import Attack, FastGradientMethod
+from cleverhans.utils_tf import model_loss
+
+from model import clone_variable, MLP_probs
+
+
+class ProjectedGradientDescentMethod(Attack):
```

Comment:
How is this different from the MadyEtAl attack merged yesterday?

Since most of the attacks are some form of projected gradient descent, the name should be more specific to the Madry Lab Challenge

Reply:
Fixed in 460d3f3. The name changed to MadryEtAlMultGPU.